QModManager/Utility/Dialog.cs

```diff
@@ -20,30 +20,30 @@ internal class Button
             internal readonly Action Action = null;
 
             internal static readonly Button Disabled = new Button();
-            internal static readonly Button SeeLog = new Button("See Log", () =>
-            {
-                string logPath;
-
-                if (Patcher.CurrentlyRunningGame == QModGame.Subnautica)
-                {
-                    logPath = Path.Combine(Patcher.QModBaseDir, "../Subnautica_Data/output_log.txt");
-                }
-                else
-                {
-                    logPath = Path.Combine(Application.persistentDataPath, "output_log.txt");
-                }
-
-                Logger.Debug($"Opening log file located in: \"{logPath}\"");
-
-                if (File.Exists(logPath))
-                {
-                    Process.Start(logPath);
-                }
-                else
-                {
-                    Logger.Error($"Expected log file at: \"{logPath}\" but none was found.");
-                }
-            });
+            //internal static readonly Button SeeLog = new Button("See Log", () =>
+            //{
+            //    string logPath;
+
+            //    if (Patcher.CurrentlyRunningGame == QModGame.Subnautica)
+            //    {
+            //        logPath = Path.Combine(Patcher.QModBaseDir, "../Subnautica_Data/output_log.txt");
+            //    }
+            //    else
+            //    {
+            //        logPath = Path.Combine(Application.persistentDataPath, "output_log.txt");
+            //    }
+
+            //    Logger.Debug($"Opening log file located in: \"{logPath}\"");
+
+            //    if (File.Exists(logPath))
+            //    {
+            //        Process.Start(logPath);
+            //    }
+            //    else
+            //    {
+            //        Logger.Error($"Expected log file at: \"{logPath}\" but none was found.");
+            //    }
+            //});
```

Comment:
Why was this removed?

Reply:
Because the dialog popup was throwing exceptions itself whenever it had to pop up, 
and the log file is no longer in this location.

I would ask you to fix this part since you know it better than I do.